### DISCUSSION TITLE
How to enlarge icons on a 4K Monitor?

### DISCUSSION THREAD

**Discussion by aisbergde** [dbeaver/dbeaver#11627] [Q&A] [answered]
The icons are very small and not so easy to view. Are there any settings to enlarge them a bit?

I am using a 4K monitor. And I am an old white man with old eyes. I even wear different glasses to be able to work reasonably on the monitor. It is a bit exhausting to decipher the small icons. I often have to bend forward to do this. The solution is not to change something in the Windows settings. I get along fine with other applications and want to leave it as it is.

**Comment by kAlvaro** [comment, marked as answer]
The usual solution in these cases (right click on "dbeaver.exe", go to the Properties/ Compatibility and turn on System (Enhanced) DPI scaling) doesn't seem to render noticeable effects for me but you might be luckier than I am.

**Comment by aisbergde** [comment]
@kAlvaro Thank you! I think this was the right idea. When I use this and set it to use System-Settings, then I get bigger Icons.
